content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -416,10 +430,10 @@ in sync.
 
 ### Docker Engine {#docker}
 
-Docker Engine is the container runtime that started it all. Formerly known just as Docker,
-this container runtime is available in various forms.
-[Install Docker Engine](https://docs.docker.com/engine/install/) explains your options
-for installing this runtime.
+On each of your nodes, install Docker for your Linux distribution as per
+[Install Docker Engine](https://docs.docker.com/engine/install/#server).
+You can find the latest validated version of Docker in this
```

Comment:
I don't think Kubernetes validates against Docker anymore, and there doesn't seem to be a version in that file.